src/components/Tags/MultiFilterStyleTagSelector.tsx

```
@@ -150,54 +150,85 @@ export function MultiFilterStyleTagSelector({
+      {tags?.map((tag) => (
+        <div
+          key={tag.id}
+          className="flex items-center gap-3 p-3 rounded-lg hover:bg-gray-50 cursor-pointer"
+          onClick={() => {
+            if (tag.has_children) {
+              handleMobileGroupClick(tag);
+            } else {
+              handleSelect(tag);
+            }
+          }}
+        >
```

Biome (2.3.14)

[error] Static Elements should not be interactive.

To add interactivity such as a mouse or key event listener to a static element, give the element an appropriate role value.

(lint/a11y/noStaticElementInteractions)